CHANGELOG.md

```diff
@@ -18,15 +21,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Changed
 
+- Projects tab layout (#447)
 - Refactored `NewComponentButton` (#450)
 - Modal styling (#450)
-- Projects tab layout (#447)
 
 ### Fixed
 
 - CookieBot consent request in iframes (#437)
 - Resetting name error if file creation/renaming is cancelled (#450)
 - Mission Zero performance gains (#457)
+- Modal button alignment support (#460)
```

Comment:
Probably missing something, but Is this meant to be in this PR? (and on line 24)

Reply:
Yes, let me fix it...